Client/src/App.js

```diff
@@ -1,12 +1,31 @@
 import React from "react";
+import Home from "./pages/Home";
+import Profile from "./pages/Profile";
+import Saved from "./pages/Saved";
+import Explore from "./pages/Explore";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 
-function App() {
+const App = () => {
   return (
-    <div>
-      <h2>Welcome to React App</h2>
-      <h3>Date : {new Date().toDateString()}</h3>
+    <div className="bg-gray-200 w-full min-h-screen">
+      <Router>
+        <Switch>
+          <Route exact path="/">
+            <Home />
+          </Route>
+          <Route path="/profile">
```

Comment:
you can also pass component prop inside your <Route/> declaration see below example. Read this https://reactrouter.com/web/api/Route
```suggestion
          <Route path="/profile" component={Profile}>
```